dvc/command/commit.py

```diff
@@ -21,7 +24,7 @@ def run(self):
                     force=self.args.force,
                 )
             except DvcException:
-                logger.error(
+                logger.exception(
                     "failed to commit{}".format(
```

Comment:
The string formatting routines shouldn't be used in the logger method invocations. This should be rewritten with:

```
"failed to commit%s", (" " + target) if target else ""
```

Reply:
totally, @ei-grad ; I delayed this change because the whole code base is currently using the brace style: `"hello {}".format("world")`, sadly, on 2.7 the logger doesn't support braces, but it does on 3.x maybe we should wait a little bit to embrace this change.